server.go

```diff
@@ -526,6 +526,11 @@ func notifySignatureComplete(signature *types.UserSignature) (err error) {
 	smtpPassword := os.Getenv(envSmtpPassword)
 	notificationAddress := os.Getenv(envNotificationAddress)
 
+	if smtpHost == "" || smtpPort == "" || notificationAddress == "" {
```

Comment:
Minor thought: Would gain slightly better code coverage by moving this check down to line 533.

Just helps to execute "more" code before exiting the test, but still doesn't require us to setup/teardown a bunch of env vars, etc.

Reply:
I just ❤️ your feedback @bhamail !